### PR TITLE
Sherllin t patch 1

### PR DESCRIPTION
**Pull request by SherllinT**

### CODE REVIEW

File: lesson_01/ReadMe.html
Comment (anthonydmays) [inline lesson_01/ReadMe.html:1]: This file is in the wrong location. Please move to a sub-folder with your name. Happy to approve once moved.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Comment by anthonydmays** [comment]
I see that you have PR #33 with the same README. You can close this PR and use that one for your work.

File: lesson_01/ReadMe.html
Comment (SherllinT) [reply lesson_01/ReadMe.html:1]: Noted, I have resolved and re-submitted in correct folder!

File: lesson_01/Sherllin/Readme.HTML
Comment (anthonydmays) [inline lesson_01/Sherllin/Readme.HTML:1]: Your version is missing the bottom borders for the title. Please fix.

**Review by anthonydmays — CHANGES_REQUESTED** [review]

**Review by anthonydmays — APPROVED** [review]

**Pull request merged**